src/mpid/ch4/src/ch4_impl.h

```diff
@@ -962,28 +962,24 @@ static inline int MPIDI_CH4U_allocate_shm_segment(MPIR_Comm * shm_comm_ptr,
                                                   void **base_ptr)
 {
     MPIR_Errflag_t errflag = MPIR_ERR_NONE;
-    int mpi_errno = MPI_SUCCESS;
+    int mpi_errno = MPI_SUCCESS, mpl_err = -1;
```

Comment:
Can we initialize `mpl_err = 0` to be consistent with `mpi_errno ` initialization ?